docker/Dockerfile

```diff
@@ -195,6 +195,12 @@ ENV USER=${USER}
 ARG HOME
 ENV HOME=${HOME}
 
+# Install curl for healtchecks
+RUN apt-get -qq update && \
+    apt-get -qq install -y --no-install-recommends \
```

Comment:
Is the flag meant to be `--quiet`? I don't see a `-qq` flag in the `apt-get` reference. Using the long name of the flag might be nice.



```suggestion
RUN apt-get -q update && \
    apt-get -q install -y --no-install-recommends \
```